docker-entrypoint.sh

```diff
@@ -23,4 +23,27 @@ if [[ "$LOG_LEVEL" ]]; then
     fi
 fi
 
+if [[ "$SSL" == "true" ]]; then
+
+    echo "In your /etc/hosts file on Linux, OS X, or Unix (with root permissions), \n edit the line of localhost so it looks like this: \n 127.0.0.1      localhost s3.scality.test"
+
+    ## Generate SSL key and certificates
+    # Generate a private key for your CSR
+    openssl genrsa -out ca.key 2048
+    # Generate a self signed certificate for your local Certificate Authority
+    openssl req -new -x509 -extensions v3_ca -key ca.key -out ca.crt -days 99999  -subj "/C=US/ST=Country/L=City/O=Organization/CN=scality.test"
+    # Generate a key for S3 Server
+    openssl genrsa -out test.key 2048
+    # Generate a Certificate Signing Request for S3 Server
+    openssl req -new -key test.key -out test.csr -subj "/C=US/ST=Country/L=City/O=Organization/CN=*.scality.test"
+    # Generate a local-CA-signed certificate for S3 Server
+    openssl x509 -req -in test.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out test.crt -days 99999 -sha256
+
+    ## Update S3Server config.json
+    # This condition makes sure that certFilePaths section is not added twice. (for docker restart)
```

Comment:
Yes, we should :) - It depends if we want or not generating new certificate when we restart the server. Generating new certificate each time might be confusing and I don't see any security issue on keeping the same one when restarting the server.